app/Services/CampaignService.php

```diff
@@ -40,9 +40,11 @@ public function find($id)
             $campaign = $this->phoenix->getCampaign($id);
 
             $this->cache->store($campaign['data']['id'], $campaign['data']);
+
+            $campaign = $campaign['data'];
         }
 
-        return $campaign['data'];
+        return $campaign;
```

Comment:
@sbsmith86 I was seeing that when it was stored in cache already, the data lived in `$campaign` and not in `$campaign['data']`, so that's why I made that change